src/backend/access/aocs/aocsam.c

```diff
@@ -910,6 +911,12 @@ aocs_insert_values(AOCSInsertDesc idesc, Datum *d, bool *null, AOTupleId *aoTupl
 								   RelationGetRelationName(idesc->aoi_rel));	/* tableName */
 #endif
 
+	/*
+	 * generate new transaction id if necessary, so dependent entities, such as
```

Comment:
All the code around starts the sentence from the capital letter and finishes with a dot. I suggest to follow this style.